.mailmap

```diff
@@ -206,11 +224,16 @@ Koan-Sin Tan <koansin.tan@gmail.com>
 Korey Sewell <ksewell@umich.edu>
 Krishnendra Nathella <Krishnendra.Nathella@arm.com> Krishnendra Nathella <krinat01@arm.com>
 ksco <numbksco@gmail.com>
-kunpai <kunpai@ucdavis.edu>
+Kunal Pai <kunpai@ucdavis.edu> Kunal Pai <62979320+kunpai@users.noreply.github.com>
+Kunal Pai <kunpai@ucdavis.edu> kunpai <kunpai@ucdavis.edu>
+Kunal Pai <kunpai@ucdavis.edu> paikunal <kunpai@ucdavis.edu>
+Kunal Pai <kunpai@ucdavis.edu> KUNAL PAI <kunpai@ucdavis.edu>
 Kyle Roarty <kyleroarty1716@gmail.com> Kyle Roarty <Kyle.Roarty@amd.com>
 Laura Hinman <llhinman@ucdavis.edu>
 Lena Olson <leolson@google.com> Lena Olson <lena@cs.wisc,edu>
```

Comment:
233 and 234 are identical

Reply:
They are not identical. One is ",edu" and the other one is ".edu".

Reply:
The ",edu" one is a typo then, as ",edu" is not a valid email address.

Reply:
Right... again, that's the point of this file is to fix these "typos" in the commits. We can't change the commit information, but we can give git a way to know that the email `lena@cs.wisc.edu` and `lena@cs.wisc,edu` are actually the same person, which is also the same as `leolson@google.com`

Reply:
I think our messages are just flying past one another, but just acknowledging you answered this with your previous post.

Reply:
No worries. Thanks for the careful review!